src/step_definitions/elements.js

```diff
@@ -304,7 +304,100 @@ defineSupportCode(function({ When, Then }) {
       .then(function(matcherResult) {
         return expect(matcherResult).to.not.be.true;
       })
-      .catch(() => Promise.resolve());
+      .catch(error => {
+        if (error.message === 'expected true to be false') {
+          return Promise.reject(error);
+        }
+        return Promise.resolve();
+      });
+  });
+
+  Then(/^there is element "([^"]*)" containing "([^"]*)" text$/, function(elementName, value) {
+    const pageElement = this.currentPage[elementName];
+
+    return this.currentPage.waitForVisibilityOf(elementName).then(() => {
```

Comment:
Nearly the same code is used for 8 steps e.g.:
- there is element
- there is no element
- there is element containing text
- there is NO element containing text
- etc.

We could add an extra method and just reuse it, in the same way as `checkNumberOfElements` method.